x/mock/simulation/transition_matrix.go

```diff
@@ -5,12 +5,11 @@ import (
 	"math/rand"
 )
 
-// TransitionMatrix is _almost_ a left stochastic matrix.
-// It is technically not one due to not normalizing the column values.
-// In the future, if we want to find the steady state distribution,
-// it will be quite easy to normalize these values to get a stochastic matrix.
-// Floats aren't currently used as the default due to non-determinism across
-// architectures
+// TransitionMatrix is _almost_ a left stochastic matrix.  It is technically
+// not one due to not normalizing the column values.  In the future, if we want
```

Comment:
Can this change be reverted, it is standard for comments to have a newline at every punctuation symbol.

Reply:
meh I think it's fine, seems clean to read this way too - we save on a line of code